src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThread.java

```diff
@@ -231,7 +231,7 @@ void fireCompletionHandlers(Outcome o) {
     /**
      * Finds the next younger {@link CpsThread} that shares the same {@link FlowHead}.
      *
-     * Can be {@code this.}
+     * Cannot be {@code this}.
```

Comment:
Did not wind up using this function; another comment, at its sole call site, notes that it is probably wrong to use it there, too (but leaving that for another day).